src/main/java/singly_linked_list/Controller.java

```diff
@@ -0,0 +1,115 @@
+package singly_linked_list;
+import java.util.Scanner;
+
+
+public class Controller {
+    private Scanner scanner;
+    private ServiceOperations serviceOperations;
+    private Node head;
+
+    public Controller() {
+        scanner = new Scanner(System.in);
+        serviceOperations = new ListService();
+    }
+
+    public void userInterface() {
+        int choose;
+        System.out.println("1-LIFO list\n2-FIFO list\n");
+        choose = scanner.nextInt();
+        if (choose == 1) {
+            System.out.println("Enter a sequence of numbers: (input ending - number 0)");
+            head = serviceOperations.createLifo();
+        } else if (choose == 2) {
+            System.out.println("Enter a sequence of numbers: (input ending - number 0)");
+            head = serviceOperations.createFifo();
+        } else {
+            System.out.println("Incorrect number.");
+            return;
+        }
+        print();
+        while (true) {
+            System.out.println("What do you want to do:\n1-Delete item;\n2-Add element BEFORE;\n3-Add element AFTER;" +
+                    "\n4-Show all elements of a sequence;\n5-Is the list sorted in ascending order?\n" +
+                    "6-Is the list sorted in descending order?\n7-Print list recursively;\n8-Exit;");
```

Comment:
Could you please add ability to create new list during program execution?

Reply:
Done.